src/main/java/org/radarcns/util/EmailSenders.java

```diff
@@ -0,0 +1,85 @@
+package org.radarcns.util;
+
+import org.radarcns.config.MonitorConfig;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+
+/**
+ * Class to store {@link EmailSender} associated with each project.
+ */
+public class EmailSenders {
+
+    private Map<String,EmailSender> emailSenderMap;
+    private IOException exception;
+
+    /**
+     * Default Constructor
+     */
+    public EmailSenders() {
```

Comment:
Is this used outside of tests? Instead, you could also make a protected `EmailSenders(Map<String, EmailSender>)` function to create custom email senders with a final emailSenderMap.

Reply:
Removed the default constructor and added - 

```java
    private final Map<String,EmailSender> emailSenderMap;

    public EmailSenders(Map<String, EmailSender> map) {
        this.emailSenderMap = map;
    }
```

Cannot make it protected because its required in tests (different package)